docs/installation/_index.md

```diff
@@ -52,69 +41,131 @@ If you are using a single board computer (SBC) or a microcontroller, prepare you
 
-Stable:
+1. Follow the steps on the **Setup** tab to install `viam-server` on your Linux computer.
+
+1. Once `viam-server` is installed and running, return to the **Setup** page on the [Viam app](https://app.viam.com) and wait for confirmation that your computer has successfully connected.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Local installation
+
+If desired, you can also install and run `viam-server` locally without connecting to the [Viam app](https://app.viam.com).
+This is useful for situations where you have limited internet access, or if your robot will never connect to the internet itself.
+You can install `viam-server` in this fashion to a board (SBC), macOS computer, or Linux computer.
+Select the tab below appropriate for your installation:
+
+{{< tabs name="Local Installation" >}}
+{{% tab name="Board (SBC)" %}}
+
+`viam-server` is distributed for Linux as an [AppImage](https://appimage.org/).
+The AppImage is a single, self-contained binary that runs on 64-bit Linux systems running the `aarch64` or `x86_64` architectures, with no need to install any dependencies.
+To install `viam-server` on a board:
+
+1. If you haven't already, you must install a supported operating system on your board.
+   Follow the instructions to [prepare your board](#prepare-your-board) to install a supported OS on your board.
+
+1. Next, `ssh` to your board.
+   Create your robot's [local configuration file](/appendix/configuration-file/), and save the file to <file>/etc/viam.json</file>.
+
+1. Run `uname -m` to determine the architecture of your board.
+   `viam-server` supports the `aarch64` and `x86_64` architectures.
+
+<!-- The below has to be in HTML because we're using a table inside another table with indentation-->
+<ol start="4">
+<li>Select the tab below matching your board's architecture, and run the command listed to download and install <code>viam-server</code> on your board.
+   We recommend the stable release for most users:
+
+{{< tabs name="linux-board-architectures" >}}
+{{% tab name="aarch64" %}}
+
+**Stable:**
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-stable-aarch64.AppImage -o viam-server &&
   chmod 755 viam-server && sudo ./viam-server --aix-install
 ```
 
-Latest:
+**Latest:**
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-latest-aarch64.AppImage -o viam-server &&
   chmod 755 viam-server && sudo ./viam-server --aix-install
 ```
 
 {{% /tab %}}
-{{% tab name="X86_64"%}}
+{{% tab name="x86_64" %}}
 
-Stable:
+**Stable:**
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 curl https://storage.googleapis.com/packages.viam.com/apps/viam-server/viam-server-stable-x86_64.AppImage -o viam-server &&
   chmod 755 viam-server && sudo ./viam-server --aix-install
```

Comment:
```suggestion
  chmod 755 viam-server
```
Can we take this bit off? Especially for x86_64?

Reply:
(Because for laptops/desktops as opposed to SBCs, it's unlikely that you'd want viam-server to be a system service; you'd likely want to run it manually only as needed)